cmd/kubeadm/app/cmd/upgrade/common.go

```diff
@@ -227,7 +227,7 @@ func getWaiter(dryRun bool, client clientset.Interface) apiclient.Waiter {
 // InteractivelyConfirmUpgrade asks the user whether they _really_ want to upgrade.
 func InteractivelyConfirmUpgrade(question string) error {
 
-	fmt.Printf("[upgrade/confirm] %s [y/N]: ", question)
+	fmt.Printf("[upgrade/confirm] %s [y/N]:\n", question)
```

Comment:
we should omit this change, because the user should write the `y` or `n` on the same line as the prompt.

Reply:
I have neglected. It has been updated. Is there anything else that needs to be updated?